.github/workflows/checks.yml

```diff
@@ -76,9 +76,9 @@ jobs:
 
       - name: Install sncast 0.6.0
         run: |
-          curl --proto '=https' --tlsv1.2 -LsSf https://raw.githubusercontent.com/foundry-rs/starknet-foundry/master/scripts/install.sh | sh -s -- -v 0.6.0
-          source /Users/runner/.bashrc || true
-          echo "$(dirname $(which sncast))" >> $GITHUB_PATH    
+          asdf plugin add starknet-foundry
+          asdf install starknet-foundry 0.6.0
+          asdf global starknet-foundry 0.6.0
```

Comment:
let's use:

- snfoundry gh action
- newe(r/st) version of snfoundry

Reply:
The first version that outputs `snforge x.y.z` instead of `forge x.y.z` is 0.9.1, meaning every version before that is incompatible with gh action.

And snforge 0.9.1 is breaking and will require rewriting some devnet code. I would rather avoid complicating this PR and migrate to 10.0.2+ when it's out (assuming declare is fixed by then), since this is not priority and both setup-snfoundry and asdf approach take 2 seconds.

Reply:
what do you mean by breaking exactly?

Reply:
Can you elaborate a bit more on it? perhaps I miss some importatnt piece of information here, but I don't think anything in cast is this breaking

Reply:
https://github.com/software-mansion/starknet.swift/pull/114
Like I said, some changes to commands outputs, not much, but be changed when removing workaround is possible.

Reply:
ok but it should be possible now, right? this change is already included in 0.10.1 if I remember correctly

Reply:
I can add changes to deserialization now, but removing workaround will have to come later as declare is not working properly as of 0.10.1

Reply:
ok, so you mean declaring multiple contracts as one package, and not commands outputs? 
Yes this will be a part of 0.11 probably. if this is blocking, then sure - let's wait.